.changelog/1772.changed.txt

```diff
@@ -0,0 +1 @@
+chore: Prepare for release v0.127.0
```

Comment:
This changelog will be - `chore: Upgrading otel collector to 0.127.0 with all the changes.`